templates/repo/issue/view_content/pull.tmpl

```diff
@@ -375,10 +375,7 @@
 									{{.CsrfTokenHtml}}
 									<input type="hidden" name="head_commit_id" value="{{.PullHeadCommitID}}">
 									<div class="field">
-										<input type="text" name="merge_title_field" value="{{.Issue.PullRequest.GetDefaultMergeMessage $.Context}}">
-									</div>
-									<div class="field">
-										<textarea name="merge_message_field" rows="5" placeholder="{{$.i18n.Tr "repo.editor.commit_message_desc"}}">Reviewed-on: {{$.Issue.HTMLURL}}&#13;&#10;{{$approvers}}</textarea>
+										<input type="text" name="merge_title_field" value="{{.DefaultMergeMessage}}">
```

Comment:
Need to confirm, is it correct and done by purpose?

Here, the `merge_message_field` was deleted?

Reply:
Accident event. See #19657